livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/tts.py

```diff
@@ -271,11 +275,14 @@ def close(self) -> None:
 def _create_speech_synthesizer(
     *, config: _TTSOptions, stream: speechsdk.audio.AudioOutputStream
 ) -> speechsdk.SpeechSynthesizer:
-    speech_config = speechsdk.SpeechConfig(
-        subscription=config.speech_key,
-        region=config.speech_region,
-        speech_recognition_language=config.language or "en-US",
-    )
+    if config.speech_host:
+        speech_config = speechsdk.SpeechConfig(host=config.speech_host)
+    else:
+        speech_config = speechsdk.SpeechConfig(
+            subscription=config.speech_key,
+            region=config.speech_region,
+            speech_recognition_language=config.language or "en-US",
+        )
```

Comment:
When there is a custom host, we don't need to configure the other fields?

Reply:
Yes, it only needs `speech_config = speechsdk.SpeechConfig(
    host="http://localhost:5000")` (ref https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-container-ntts?tabs=container&pivots=programming-language-python#use-the-container)